common/util_test.go

```diff
@@ -20,7 +20,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-package common
+package common_test
```

Comment:
I'm generally in favor of external tests, so this is not a complaint, but: is there a motivation reason for this change?

with the copy/pasted hardcoded values, it feels like a bit of an odd separation.  though I don't think those are useful tests anyway.

Reply:
The motivation is circular dependencies with our common package, dynamic config, and logger, which prevents us from using it.

Reply:
I'll revert this change. I can go with log.NewNoop(). I'll try to craft a separate PR that separates throttledLogger from loggerImpl to break the cycle.